fastlane/Fastfile

```diff
@@ -27,5 +27,93 @@ platform :android do
   lane :check do
     gradle(task: "check")
   end
+
+  desc "Takes a release tag and updates changelog"
+  lane :tag_release do
+
+    UI.user_error!("You need to provide a Github Personal token via a `GITHUB_TOKEN` environment variable - you can create a new token on Github https://github.com/settings/tokens/new") unless !ENV["GITHUB_TOKEN"].nil?
+
+    # ensure repo is clean and on master
+    ensure_git_status_clean
+    ensure_git_branch
+
+    # pull latest changes and fetch tags
+    git_pull
+
+    # get current version
+    current_tag = last_git_tag
+    version = Semantic::Version.new(current_tag)
+
+    # prompt for bump type
+    bump_type = UI.select(
+      "Current version is #{current_tag}. What would you like to bump?",
+      ["major", "minor", "patch"]
+    )
+
+    # increment version
+    case bump_type
+    when "major"
+      tag = version.increment!(:major)
+    when "minor"
+      tag = version.increment!(:minor)
+    else
+      tag = version.increment!(:patch)
+    end
+
+
+    sh("git", "checkout", "-b", "tag/#{tag.to_string}")
```

Comment:
Im not 100% sure myself, but just want to check this is the correct syntax?

Currently in android-app repo, we have it in one quote like this: 
`sh("git checkout -b release/#{last_git_tag}")`

Reply:
Odd. I was following the fastlane docs 🤔